javagen/src/main/java/com/azure/autorest/util/returntype/ReturnTypeDescriptionAssembler.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+
+package com.azure.autorest.util.returntype;
```

Comment:
Do we need a separate package specific to `returntype`?

Reply:
I'll move it to an existing package in the next PR.